packages/utils/src/cmds/transform_translate.js

```diff
@@ -0,0 +1,381 @@
+/**
+ * Example usage:
+ *
+ * d2-utils transform_translate \
+ *   --in-dir ~/development/dhis2/maintenance-app/src/i18n \
+ *   --out-dir ~/development/dhis2/project-doom/i18n \
+ *   --override-existing-files \
+ *   --app-name maintenance-app \
+ *   --languages fr,ar \
+ *   --delete-old-files \
+ *   --log-missing-keys
+ */
+const path = require('path')
+const fs = require('fs')
+const log = require('@dhis2/cli-helpers-engine').reporter
+
+const CONSUMING_ROOT = path.join(process.cwd())
```

Comment:
It might be worth allowing the cwd to be passed in as an argument?

Reply:
That'd be redundant as you can do that already with `inDir` and `outDir`